src/qml/controls/PeerRadioButton.qml

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import org.bitcoincore.qt 1.0
+
+RadioButton {
```

Comment:
> This commit does not include the disconnect/ban buttons, the top right buttons to cycle through the peers, or the icons that are adjacent to the values.

We should remove this file then and the references in bitcoin_qml.qrc and the makefile

Reply:
Yes, I missed removing these ones from this commit